addons/web/static/src/views/graph/graph_controller.xml

```diff
@@ -38,6 +38,10 @@
                 t-on-click="toggleStacked"
                 t-att-class="{ active: model.metaData.stacked }"
             />
+            <button class="btn btn-secondary fa fa-signal o_graph_button" data-tooltip="Cumulative" aria-label="Cumulative"
+                t-on-click="toggleCumulated"
+                t-att-class="{ active: model.metaData.cumulated }"
```

Comment:
why display the button if you can't click it? t-if = model.metaData.cumulated ?

Reply:
active is not the same as clickable @FrancoisGe ? It just change its style to underline it's active